main.cpp

```diff
@@ -56,19 +56,17 @@ struct Comparison
 struct U
 {
     float num1 { 0 }, num2 { 0 };
-    float updateNums(float* updated)      //12
+    float updateNums(const float& updated)      //12
     {
-        if(updated != nullptr)
+        //if(updated != nullptr)
         {
```

Comment:
Don't just comment it out.  Delete the curly braces as well and fix the indentation
- [ ] revise